app/controllers/admin/users_controller.rb

```diff
@@ -34,8 +30,20 @@ def destroy
 
     private
 
+    def sort_column
+      User.column_names.include?(params[:sort]) ? params[:sort] : 'username'
+    end
+
+    def sort_direction
```

Comment:
Do you think we can use this method in another controller as well?
So we can move it to concern. Thoughts?